gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -497,12 +502,22 @@ def prepare_ht_for_validation(
     )
 
     logger.info("Constructing INFO field")
+    # Remove SIFT and Polyphen from csq fields or they will be inserted with
+    # missing values by vep_struct_to_csq
```

Comment:
Do you want to add a note stating they're extracted from VEP fields and processed separately?